ethcore/src/client/client.rs

```diff
@@ -141,6 +142,19 @@ pub struct Client {
 }
 
 const HISTORY: u64 = 1200;
+// database columns
+/// Meta(?)
+pub const DB_COL_META: Option<u32> = Some(0);
```

Comment:
I've added this for meta-DB values, such as version. Can be removed if it is not used.